scanpy/preprocessing/_highly_variable_genes.py

```diff
@@ -455,12 +525,17 @@ def highly_variable_genes(
             "pass `inplace=False` if you want to return a `pd.DataFrame`."
         )
 
-    if flavor == "seurat_v3":
+    if (
+        flavor == "seurat_v3"
+        or flavor == "seurat_v3_paper"
+        or flavor == "seurat_v3_implementation"
+    ):
```

Comment:
```suggestion
    if flavor in {"seurat_v3", "seurat_v3_paper", "seurat_v3_implementation"}:
```

Reply:
Nice forgot to revisit this before the commit thx!